### PR TITLE
[CNSMR-3226] - Application Distribution documentation

### PR DESCRIPTION
**Pull request by zopa-simon-cass**
Ticket: https://babylonpartners.atlassian.net/browse/CNSMR-3226

### Why?

More documentation is always a good thing! https://babylonhealth.slack.com/archives/GNYBQ06P5/p1575449885038000 

### PR checklist
* [x] I've assigned this PR to myself
* [x] I've set all the necessary PR's labels
* [] If I touched files in the SDK, I've updated `SDK/CHANGELOG.md` and added `#SDK` in the PR title

With :heart:

### CODE REVIEW

File: Cookbook/Technical-Documents/ApplicationDistribution.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Application Distribution
+This document is for any Babylon iOS developers who wish to distribute our applications, or for any partners who wish to gain access to our applications.
```
Comment (viorel15) [inline Cookbook/Technical-Documents/ApplicationDistribution.md]:
```suggestion
This document is for Babylon iOS developers for distribution of our applications internally, to other Babylon employees, or externally, to our partners.
 ```

File: Cookbook/Technical-Documents/ApplicationDistribution.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## How to gain access to iOS applications
+### External Babylon partners
+* If you are an external partner of Babylon, please speak with your Babylon contact who will arrange this for you
+
+### Babylon Employees
+* If you are a Babylon employee, please reach out to the current iOS support engineer(s) in the #ios slack channel
+* You will also need to have [App Center](https://github.com/babylonhealth/ios-playbook/blob/simon/cnsmr-3226/Cookbook/Technical-Documents/AppCenter.md) installed
```
Comment (viorel15) [inline Cookbook/Technical-Documents/ApplicationDistribution.md]: This document should not be intended for anyone other than iOS developers.

Instead I would focus on the process form this point of view:
```
 Requests to grant access must be through #ios channel and it must list each app they need access for. 
```
 You should list here which apps are eligible for each group. Keep in mind that Babylon meplyeees can have access to Testflight if they need to.

File: Cookbook/Technical-Documents/ApplicationDistribution.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Distribution of iOS applications
+### Distributing to a Babylon employee
```
Comment (viorel15) [inline Cookbook/Technical-Documents/ApplicationDistribution.md]:
```suggestion
### Distributing to AppCenter
```

File: Cookbook/Technical-Documents/ApplicationDistribution.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+* Ask the person who is requesting access for their `@babylonhealth.com` email address
+* Follow [this guide](https://github.com/babylonhealth/ios-playbook/blob/simon/cnsmr-3226/Cookbook/Technical-Documents/AppCenter.md#adding-people-to-the-organisation) to invite them to our App Center builds
+
+### Distributing to an external partner
```
Comment (viorel15) [inline Cookbook/Technical-Documents/ApplicationDistribution.md]:
```suggestion
### Distributing to TestFlight
```

**Review by viorel15 — CHANGES_REQUESTED** [review]
Left few comments you should address before approving. Otherwise it's good

File: Cookbook/Technical-Documents/ApplicationDistribution.md (hunk 1 above)
Comment (scass91) [reply Cookbook/Technical-Documents/ApplicationDistribution.md]: I think this reads better with your suggestion, keeping the `who wish` part in it - what do you think?

Comment (viorel15) [reply Cookbook/Technical-Documents/ApplicationDistribution.md]: sure. That is ok

File: Cookbook/Technical-Documents/ApplicationDistribution.md (hunk 3 above)
Comment (scass91) [reply Cookbook/Technical-Documents/ApplicationDistribution.md]: I'm going to rename this to `Distributing via App Center`

File: Cookbook/Technical-Documents/ApplicationDistribution.md (hunk 4 above)
Comment (scass91) [reply Cookbook/Technical-Documents/ApplicationDistribution.md]: As above, will change this to `Distributing via Testflight`

File: Cookbook/Technical-Documents/ApplicationDistribution.md (hunk 2 above)
Comment (scass91) [reply Cookbook/Technical-Documents/ApplicationDistribution.md]: @viorel15 by `which apps are eligible for each group` can you explain further what you mean by group here please? Is the `group` here relating to internal vs external?

Comment (viorel15) [reply Cookbook/Technical-Documents/ApplicationDistribution.md]: Yes, that is what I meant. Employees vs partners

Comment (scass91) [reply Cookbook/Technical-Documents/ApplicationDistribution.md]: I've made some changes - please let me know your thoughts @viorel15

**Review by viorel15 — APPROVED** [review]
👍

**Review by mluisbrown — APPROVED** [review]

**Pull request merged**